spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestDataFrameWrites.java

```diff
@@ -220,16 +220,15 @@ private void writeDataWithFailOnPartition(
     final int numPartitions = 10;
     final int partitionToFail = new Random().nextInt(numPartitions);
     MapPartitionsFunction<Row, Row> failOnFirstPartitionFunc =
-        (MapPartitionsFunction<Row, Row>)
```

Comment:
We can fix it for other version of spark folders also.

Reply:
Similar comment for all Spark and Flink module related changes. Only default modules are handled currently. Maybe follow up PR also is fine.